docs/ops/cli.md

```diff
@@ -265,6 +268,9 @@ Action "run" compiles and runs a program.
                                           connect to a different JobManager than
                                           the one specified in the
                                           configuration.
+     -sae,--schutdownOnAttachedExist      If the job mode is not detached and
+                                          cluster has been started for the job,
+                                          perform the best effort cluster.
```

Comment:
I think there is something missing in the description: _[...] perform the best effort cluster._